proto/src/determined/api/v1/trial.proto

```diff
@@ -489,6 +489,8 @@ message AllocationPendingPreemptionSignalRequest {
   };
   // The id of the allocation.
   string allocation_id = 1;
+  // The resource manager for the request.
+  string resource_manager = 2;
```

Comment:
for both changes in this file, let's not make the python code know which RM name we need and just figure it out from the allocation id instead.

Reply:
This applies to dispatcherrm only